src/main/java/com/lunark/lunark/properties/model/Property.java

```diff
@@ -77,6 +70,19 @@ public void setApproved(boolean approved) {
     @Transient
     private Clock clock = Clock.systemDefaultZone();
     private PropertyType type;
+    private Long hostId;
```

Comment:
If the `hostId` were just a Long we would not have the DBMS checking if its value corresponds to and id of a host in the database. I recommend changing this attribute to be of type of `Account` and adding JPA annotations. I set the name in JoinColumn to be the same as in the import script.
```suggestion
    @ManyToOne
    @JoinColumn(name = "host_id")
    private Account host;
```